lib/drs/Dockerfile.dev

```diff
@@ -0,0 +1,32 @@
+ARG BASE_IMAGE
+ARG BASE_IMAGE_VERSION
+
+FROM "${BASE_IMAGE}:${BASE_IMAGE_VERSION}"
+
+# TODO: change USER
+USER root
+
+RUN mkdir /drs && \
+    mkdir /wes;
+
+WORKDIR /drs
+
+ARG REPO
+ARG BRANCH
+ARG TAG
+
+ARG PROJECT_DIRECTORY
```

Comment:
I don't think we'll be needing this unless we plan to build out the dockerfile like we do with katsu whereby we specify the project path inside the container from the outside.